app/javascript/src/game/ui/lobby.js

```diff
@@ -1,18 +1,20 @@
 /*jshint esversion: 6 */
 export class Lobby {
-  constructor(payload, elementId="lobby") {
-    this.payload = payload;
+  constructor({players=[]}, elementId="lobby") {
```

Comment:
Players/payloads default should probably be a en empty object, rather than an array?

Reply:
Yea this looks a bit strange but it makes sense - the curly braces around the players are saying "with the object given as a parameter, get the 'players' property".
Then the '=' just gives the default value as an empty array.
See: https://www.sitepoint.com/es6-destructuring-assignment/#defaultfunctionparameters

We could also do this:
`constructor({players=[]}={}, elementId="lobby") {`
Where `{players=[]}={}` would return an empty object in the case that the object was undefined. Depends how defensive we want to program I guess.